### ISSUE TITLE
Own issues filter

### ISSUE THREAD

**Issue by altteo** [gitspeak/community#18]
It will be convinient to have a filter to see all my issues (where I'm an author and maybe where I have commented). At the moment I need to type my login int filter input

**Comment by somebee** [comment]
Good idea. Maybe we should add tabs inside/under Issues in sidebar for "Created", "Assigned", and "Mentioned"?

**Comment by altteo** [comment]
You’d better keep interface clear :-) The tasks that are common:
1. To see what concerns me
Only one filter will do. Maybe “assigned to me” also useful but it is more obvious for you (i’m not assigned through github). Maybe assignee could be put as avatar in the list of issues.
2. To see new things that are addressed to me.
Issues are just conversation topics, so maybe it is a good idea to take logic from messengers (particularly telegram). Sort by the last activity, blue dot represent new in the issue concerning me, grey dot represent update in third-party issue.

Really, think of issues as chats in the messenger. They just have pined header, assignee, labels, and could be closed.

**Comment by aalemayhu** [comment]
Filtering on assigned is possible but the author is not yet supported. Tracked in gitspeak/gitspeak#763
